doc/articles/contributing/pre-release-builds.md

```diff
@@ -0,0 +1,4 @@
+# pre-release builds
```

Comment:
🎗 needs to be added to the toc. this could be a section within an existing document.